nifi-marklogic-processors/src/test/java/org/apache/nifi/marklogic/processor/ExtensionCallMarkLogicIT.java

```diff
@@ -58,12 +56,12 @@ void simpleGet() {
 
         runner.assertQueueEmpty();
         assertEquals(0, runner.getFlowFilesForRelationship(ExtensionCallMarkLogic.FAILURE).size());
-        assertEquals(1, runner.getFlowFilesForRelationship(ExtensionCallMarkLogic.SUCCESS).size());
+        assertEquals(2, runner.getFlowFilesForRelationship(ExtensionCallMarkLogic.SUCCESS).size());
```

Comment:
You can remove this line, it shouldn't have been there before - it's redundant with the two lines below.